src/video/wayland/SDL_waylandvideo.c

```diff
@@ -495,6 +495,11 @@ static bool Wayland_IsPreferred(struct wl_display *display)
 
     wl_registry_destroy(registry);
 
+    if (!preferred_data.has_fifo_v1)
+        SDL_LogWarn(SDL_LOG_CATEGORY_VIDEO, 
+                    "Your compositor doesn't appear to support fifo-v1, which SDL needs if it wants to use Wayland by default without harming GPU-bound performance and frame pacing due to swapchain starvation. You'll be getting Xwayland instead. Set SDL_VIDEODRIVER=wayland if you're okay with this, or ask your compositor maintainers to support fifo-v1.");
```

Comment:
```suggestion
                    "Your compositor doesn't appear to support fifo-v1, which SDL needs if it wants to use Wayland by default without harming GPU-bound performance and frame pacing due to swapchain starvation. You'll be getting Xwayland instead. Set SDL_VIDEO_DRIVER=wayland if you're okay with this, or ask your compositor maintainers to support fifo-v1.");
```

Reply:
I put the environment variable here, because this would be more user-visible, than intended for *developers*.

Reply:
It's not the location, it's the spelling: in SDL3, it's `SDL_VIDEO_DRIVER`, not `SDL_VIDEODRIVER`